### ISSUE TITLE
Option to disable IP logging.

### ISSUE THREAD

**Issue by PringlePot** [mailcow/mailcow-dockerized#4170]
## Summary

I'd like it to be possible to disable ips in logs. I believe this is needed for websites.

## Motivation

This would really help sites trying to be privacy concerned, but are currently unable to use mailcow due to unencrypted ip addresses showing in logs.

**Comment by Adorfer** [comment]
more granularity in terms of (client ip) loglevel would be an appreciated enhancement.  
e.g. extreme short retention time, or no retention of ip in the main log. 
what ever is possible without bit effort.

**Comment by PringlePot** [comment]
Maybe possible make ips show as like x.x.x.x or something like that

**Comment by andryyy** [comment]
Perhaps a way to replace the IP by the ASN after n days?

**Comment by PringlePot** [comment]
or just directly?

**Comment by andryyy** [comment]
I don’t think this will work. It would be too slow. We also need some logs to ban IPs etc. 

I really want to know who connnected and changed this or that. It should be logged. Imagine getting in legal trouble. Sometimes it is absolutely worth it. 

Data should not be saved for too long though. That’s something we need to change.

**Comment by waja** [comment]
Please make the retention time configurable. Maybe others want to keep logs longer.

**Comment by PringlePot** [comment]
Any updates?
